push_notifications/models.py

```diff
@@ -1,7 +1,9 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
+from firebase_admin import messaging
```

Comment:
This import should be placed in the `gcm` package, since if placed here it turns `firebase_admin` as a mandatory requirements for everyone.